Make gem binaries on ruby 2.5 use a 25 suffix, so you can have both
other versions of the same gem installed at the same time
without conflicts.

Index: lib/rubygems/commands/install_command.rb
--- lib/rubygems/commands/install_command.rb.orig
+++ lib/rubygems/commands/install_command.rb
@@ -20,7 +20,7 @@ class Gem::Commands::InstallCommand < Gem::Command
 
   def initialize
     defaults = Gem::DependencyInstaller::DEFAULT_OPTIONS.merge({
-      :format_executable => false,
+      :format_executable => true,
       :lock              => true,
       :suggest_alternate => true,
       :version           => Gem::Requirement.default,
